abnf/odata-aggregation-abnf.txt

```diff
@@ -230,12 +230,11 @@ searchTrafo     = %s"search" OPEN BWS searchExpr BWS CLOSE
 
 groupbyTrafo    = %s"groupby" OPEN BWS groupbyList [ BWS COMMA BWS applyExpr ] BWS CLOSE
 groupbyList     = OPEN BWS groupbyElement *( BWS COMMA BWS groupbyElement ) BWS CLOSE
-groupbyElement  = groupingProperty / rollup / rollupAll
+groupbyElement  = groupingProperty / rollup / rollupDesc
 
-rollup            = %s"rollup"    OPEN BWS ( rollupUnnamedHier / rollupNamedHier ) BWS CLOSE
-rollupAll         = %s"rollupall" OPEN BWS ( rollupUnnamedHier / rollupNamedHier ) BWS CLOSE
-rollupUnnamedHier = groupingProperty 1*( BWS COMMA BWS groupingProperty )
-rollupNamedHier   = recHierQualifier
+rollup          = ( %s"rollupall" / %s"rollup" )

```

Comment:
1. This rule now defines rolling up a leveled hierarchy. I suggest to reflect this in the name, e.g. rollupLevels. 
2. This rule must include named leveled hierarchies, which are still missing here.

Reply:
1. Wouldn't a name change `rollup` -> `rollupLevels` be incompatible?
2. The named leveled hierarchies are in line 236:
https://github.com/oasis-tcs/odata-abnf/blob/0ce3356e5eef203bae4212087daa8abcd1b3af0a/abnf/odata-aggregation-abnf.txt#L235-L236